source/yellow/index.html

```diff
@@ -311,7 +311,7 @@ <h1>Open Hardware</h1>
 </div>
 
 <div
-  class="fab"
+  class="fab" style="position: fixed; top: 78px; height: 48px;"
```

Comment:
We have the same issue on https://www.home-assistant.io/skyconnect/ so I think that we should fix it for all.